imports/ui/accounts/Account.jsx

```diff
@@ -70,6 +70,7 @@ export default class AccountDetails extends Component {
             totalValueInUSD: 0,
             redelegations: [],
             hasIncentive: false,
+            operator_address : "",
```

Comment:
Why this only happens to Kava? I don't see the necessity of putting the `operator_address` here as the commission is retrieved from `getBalance` and passed to `WithdrawButton`.

https://github.com/forbole/big-dipper/blob/d506f71c6003fb6ded2a668d35dc87ff7ee5304f/imports/ui/accounts/Account.jsx#L162
https://github.com/forbole/big-dipper/blob/d506f71c6003fb6ded2a668d35dc87ff7ee5304f/imports/ui/accounts/Account.jsx#L335

Also the `WithdrawButton` only needs to know the `validator_address` as the message only take one value.

The example tx in lcd.kava.forbole.com:1317/txs/57ADDC4CF17861680643A1A2D9DC13E36FD336F204B86B2160C6CB4FD1BF8AFA

```json
      {
          "type": "cosmos-sdk/MsgWithdrawValidatorCommission",
          "value": {
            "validator_address": "kavavaloper14kn0kk33szpwus9nh8n87fjel8djx0y02c7me3"
          }
        }
```

Reply:
`Why this only happens to Kava? I don't see the necessity of putting the operator_address here as the commission is retrieved from getBalance and passed to WithdrawButton.`

The commission retrieved from getBalance returns the amount of commission, however, we also need to also pass the validator address hence this value

`Also the WithdrawButton only needs to know the validator_address as the message only take one value.
`
You're right, I will remove those unnecessary values.

Reply:
I see. I have mixed up with the variable names. The `operator_address` is actually the `validator_address`.